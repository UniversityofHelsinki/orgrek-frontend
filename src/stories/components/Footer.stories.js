import FooterComponent from '../../components/Footer';

export default {
  component: FooterComponent,
  parameters: {
    layout: 'fullscreen',
  },
};

export const Footer = {};
