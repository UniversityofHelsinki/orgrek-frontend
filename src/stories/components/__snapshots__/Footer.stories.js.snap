// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stories/components/Footer Footer smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": Array [
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "hy_logo",
      "pressed": undefined,
      "role": "img",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "university_of_helsinki",
      "pressed": undefined,
      "role": "text",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "hy_address_part1",
      "pressed": undefined,
      "role": "text",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "
",
      "pressed": undefined,
      "role": "LineBreak",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "hy_address_part2",
      "pressed": undefined,
      "role": "text",
      "value": undefined,
    },
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "hy_switchboard",
      "pressed": undefined,
      "role": "text",
      "value": undefined,
    },
  ],
  "name": "stories-components-footer--footer",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;
